translations/Readme.md

```diff
@@ -0,0 +1,30 @@
+# About Translation
+
+## How to use the translation utilities
+
+### On Windows
+
+#### 1. Adding translation `(.ts)` files
+
+1. Open `lupdate.cmd` in editer. (e.g, VSCode or Notepad, etc.)
+2. Append your new file name to the end of first line, in the format **locale_**_language-code_**.ts** (e.g, locale_de, local_zh-cn, etc.).
+3. Run the command file in cmd.
```

Comment:
(just double click it in Explorer)

Reply:
Actually these stteps 3. and 4 are the substeps of 
```
 #### 2. Updateting translation `(.ts)` files


After having added a language as per step 1, or when having modified the plgin itself

1. Just double click on `lupdate.cmd` or run it in `cmd.exe`.
2. You will find the new/updated file(s) in the same location as the `lupdate.cmd`.

#### 3. Translating the strings

Use an editor (e.g, VSCode or Notepad, etc.) or Qt's Linguist to translate the strings

#### 4. Generating / Updating `.qm` files

Just double click on `lrelease.cmd` or run it in `cmd.exe` or use Qt's Linguist's "File > Release"

```

Reply:
Adding a new translation is step 1
Updating translations is step 2 (with step 1 being optional)
Translating is step 3

Reply:
That `3. Run the command file in cmd.` should be `Just double click on `lupdate.cmd` or run it in `cmd.exe`.